bool in_rt_1 
bool in_rt_2 
float32 en_rt
bool in_lt_1 
bool in_lt_2 
float32 en_lt
